Consul/Helper/MultiLockHandler.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace SensioLabs\Consul\Helper;
+
+use SensioLabs\Consul\Services\KV;
+use SensioLabs\Consul\Services\Session;
+
+class MultiLockHandler
+{
+    /**
+     * @var array
+     */
+    private $resources;
+
+    /**
+     * @var int
+     */
+    private $ttl;
+
+    /**
+     * @var Session
+     */
+    private $session;
+
+    /**
+     * @var KV
+     */
+    private $kv;
+
+    /**
+     * @var string
+     */
+    private $sessionId;
+
+    /**
+     * @var string
+     */
+    private $lockPath;
+
+    /**
+     * @param array   $resources
+     * @param int     $ttl
+     * @param Session $session
+     * @param KV      $kv
+     * @param string  $lockPath
+     */
+    public function __construct(array $resources, $ttl, Session $session, KV $kv, $lockPath)
+    {
+        $this->resources = $resources;
+        $this->ttl = $ttl;
+        $this->session = $session;
+        $this->kv = $kv;
+        $this->lockPath = $lockPath;
+    }
+
+    /**
+     * @return bool
+     */
+    public function lock()
+    {
+        $result = true;
+
+        // Start a session
+        $session = $this->session->create('{"LockDelay":0, "TTL": "'.$this->ttl.'s"}')->json();
```

Comment:
May be it's better to use `json_encode` here. For example, `ttl` value is never controlled...